relay-dynamic-config/src/feature.rs

```diff
@@ -31,6 +31,9 @@ pub enum Feature {
     /// Enable extracting resource spans.
     #[serde(rename = "projects:span-metrics-extraction-resource")]
     SpanMetricsExtractionResource,
+    /// Enable OTel span ingestion.
+    #[serde(rename = "organizations:otel-span-ingestion")]
+    OTelSpanIngestion,
```

Comment:
nit

https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case

```suggestion
    OtelSpanIngestion,
```